studio/src/studio.js

```diff
@@ -58,19 +62,19 @@ class MasStudio extends LitElement {
         return BUCKET_TO_ENV[this.bucket] || BUCKET_TO_ENV.e59433;
     }
 
-    currentPage = new StoreController(this, Store.currentPage);
+    page = new StoreController(this, Store.page);
```

Comment:
`pageController`?

Reply:
since these are lit controller, I think we should follow it, but let's discuss it in the dev call.

Reply:
my vote goes for page, doesn't look too confusing atm